data/assets/scene/solarsystem/missions/osirisrex/actions.asset

```diff
@@ -83,6 +83,59 @@ local toggle_sun_marker = {
   isLocal = false
 }
 
+local aimAtBennu = {
+  Identifier = "os.missions.osirisrex.aimAtBennu",
+  Name = "Look at Bennu from OSIRIS-REx",
+  Command = [[
+      openspace.setPropertyValueSingle("NavigationHandler.OrbitalNavigator.RetargetAnchor", nil);
+      openspace.setPropertyValueSingle("NavigationHandler.OrbitalNavigator.Anchor", 'OsirisRex');
+      openspace.setPropertyValueSingle("NavigationHandler.OrbitalNavigator.Aim", 'Earth');
+      openspace.setPropertyValueSingle("NavigationHandler.OrbitalNavigator.RetargetAim", nil);
+      openspace.setPropertyValueSingle("NavigationHandler.OrbitalNavigator.Anchor", 'OsirisRex');
+      openspace.setPropertyValueSingle("NavigationHandler.OrbitalNavigator.Aim", 'BennuBarycenter');
```

Comment:
Do we need to do the aiming towards Earth and _then_ the aiming towards Bennu?

Reply:
I just copied things from the script log - but yeah, no we don't